src/sinks/pulsar.rs

```diff
@@ -117,6 +117,10 @@ impl SinkConfig for PulsarSinkConfig {
     fn sink_type(&self) -> &'static str {
         "pulsar"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
It looks like pulsar supports receipts when sending, so I think this should be true?
https://docs.rs/pulsar/4.1.1/pulsar/producer/struct.Producer.html#method.send

Reply:
Here again, I don't see that we either set the status in this sink, nor does it return a response status for another layer to set it. So, I don't think we do so.